docker-compose.yml

```diff
@@ -52,6 +52,11 @@ services:
       # Addresses for deployment of contracts to local blockchain will get
       # written to the build files
       - ./origin-contracts/build:/app/origin-contracts/build
+      - ./origin-js/scripts:/app/origin-js/scripts
+      - ./origin-js/src:/app/origin-js/src
```

Comment:
Only the `scripts/` directory matters here as the `origin-js` code doesn't compile at all in this container, it just runs some scripts

Reply:
Aha I see tnx